atom/browser/atom_browser_client.cc

```diff
@@ -230,19 +232,20 @@ void AtomBrowserClient::OverrideSiteInstanceForNavigation(
       content::BrowserThread::UI, FROM_HERE,
       base::Bind(&Noop, base::RetainedRef(site_instance)));
 
-  // Remember the original renderer process of the pending renderer process.
-  auto current_process = current_instance->GetProcess();
+  // Remember the original web contents for the pending renderer process.
   auto pending_process = (*new_instance)->GetProcess();
-  pending_processes_[pending_process->GetID()] = current_process->GetID();
+  pending_processes_[pending_process->GetID()] =
+      content::WebContents::FromRenderFrameHost(render_frame_host);;
```

Comment:
@zcbenz I'm a little nervous about this change, I'm not sure if there are times when we would have a pending render process host but not render frame host. Through my testing, it seemed like the render frame host for the proper web contents was passed in each time so it seemed okay to track the pending process to the originating contents.

Do you remember why initially this mapped process host to process host instead of using the frame host to track the web contents doing the navigation?